tests/integration/test_repositories.py

```diff
@@ -26,8 +26,8 @@ class RepositoriesTestCase(IntegrationTestCase):
         pipeline_iri, pipeline_id = await self.create_pipeline(location=None)
         result = await self.describe(pipeline_iri)
         self.assertTrue(result and result[pipeline_iri])
-        self.assertNotIn(SwarmUI.services, result[pipeline_iri])
-        self.assertFalse(self.project_exists(pipeline_id))
+        self.assertIn(SwarmUI.services, result[pipeline_iri])
+        self.assertTrue(self.project_exists(pipeline_id))
```

Comment:
Looks weird. The test tests the consistency of a pipeline with no repository. How could it have services if it has no repository? How could the project exists?